src/Table/Actions/Action.php

```diff
@@ -96,7 +96,7 @@ public function getName(): string
      */
     public function getModalKey(): string
     {
-        return 'action-'.strtolower($this->getKey());
+        return strtolower(Str::random());
```

Comment:
Is not it `->lower()`?
```suggestion
        return Str::random()->lower();
```